whichone/templates/feedback.html

```diff
@@ -16,53 +15,44 @@
     {% block styles -%}
     <link rel="stylesheet" type="text/css"
         href="https://cdn.rawgit.com/mfd/f3d96ec7f0e8f034cc22ea73b3797b59/raw/856f1dbb8d807aabceb80b6d4f94b464df461b3e/gotham.css">
-    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/main.css')}}">
-    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/skeleton.css')}}">
-    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/normalize.css')}}">
+    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/redesign.css')}}">
```

Comment:
What does redesign represent? Is it still the main.css?

Reply:
It's a re-written version of main.css that has way more reusable components. It still contains most of the css so we could rename it back to main